.github/workflows/test-changed-firestore-integration.yml

```diff
@@ -13,6 +13,21 @@ jobs:
       with:
         # This makes Actions fetch all Git history so run-changed script can diff properly.
         fetch-depth: 0
+    - uses: 'google-github-actions/auth@v0'
+      with:
+        credentials_json: '${{ secrets.JSSDK_ACTIONS_SA_KEY }}'
+    # create composite indexes with Terraform
+    - name: Setup Terraform 
+      uses: hashicorp/setup-terraform@v2
+    - name: Terraform Init
+      run: terraform init
+      continue-on-error: true
+    - name: Terraform Apply
+      if: github.event_name == 'pull_request'
+      run: |
+        cp config/ci.config.json config/project.json
```

Comment:
This happens below, too. Might be better to move the step on line 43 up above this step and then only have the `terraform apply` in this step.

Reply:
Done.